pyproject.toml

```diff
@@ -30,6 +30,7 @@ oresat-gen-fw-files = "oresat_configs.scripts.gen_fw_files:gen_fw_files"
 oresat-gen-dcf = "oresat_configs.scripts.gen_dcf:gen_dcf"
 oresat-print-od = "oresat_configs.scripts.print_od:print_od"
 oresat-sdo-transfer = "oresat_configs.scripts.sdo_transfer:sdo_transfer"
+oresat-gen-kaitai = "oresat_configs.scripts.gen_kaitai:gen_kaitai"
```

Comment:
I have no strong opinion one way or the other. My understanding was both were to be supported so I added kaitai as well to the list.

What were the legacy reasons exactly? I wasn't aware of any generator usage that wasn't already manual. I think it would be better to flip the switch now to one or the other so we don't have split modes and less test resulting coverage.

Reply:
It turned out oresat-firmware uses `oresat-gen-fw-files` as part of the build process. I had been waiting until sometime after OreSat0.5 handoff to not shake the tree too much in case there were other uses that I wasn't aware of. 

But yeah, I'm happy to make the change to just the script `oresat-configs` now. There's other large changes happening in oreast-firmware so it's a good time.